tox.ini

```diff
@@ -105,29 +105,7 @@ commands =
 # )
 usedevelop=true
 
-# A test suite for the oldest supported versions of Python libraries, to catch
-# any uses of APIs not available in them.
-[testenv:py3-old]
-skip_install = true
-usedevelop = false
-deps =
-    Automat == 0.8.0
-    lxml
```

Comment:
I don't see these being moved into the new bits, but maybe that's OK?

Reply:
It should be okay.

- Automat isn't a direct dependency (it's not in python_dependencies,py nor pyproject.toml), so I don't think we have any business pinning it to a particular version anyway;
- lxml will be installed [it's part of `all`](https://github.com/matrix-org/synapse/blob/dd5cc37aa465df8e33fd872de2528dc53740b2a6/pyproject.toml#L217-L227), and in the config we install `all` extras
- markupsafe is also not a direct dependency.
- the base dependencies are pip, setuptools, and stuff related to `coverage`. We've stopped using coverage (and I suspect it'd need some config fiddling to have it work in poetry anyway).